pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/SchemasResource.java

```diff
@@ -165,7 +165,7 @@ public void getSchema(
         validateDestinationAndAdminOperation(tenant, namespace, topic, authoritative);
 
         String schemaId = buildSchemaId(tenant, namespace, topic);
-        ByteBuffer bbVersion = ByteBuffer.allocate(Long.SIZE);
```

Comment:
`Long.SIZE == 64` is very very confusing..

Reply:
agreed. :(